README.md

```diff
@@ -0,0 +1,175 @@
+
+### JSON
+
+To encrypt secrets to a JSON file:
+
+```js
+const secrets = require("gitops-secrets");
+
+async function main() {
+  const payload = await secrets.providers.doppler.fetch();
+
+  // Internally managed storage
+  secrets.encryptToFile(payload);
+
+  // Custom path
+  secrets.encryptToFile(payload, { path: ".secrets.enc.json" });
+}
+
+main();
+```
+
+To decrypt secrets from a JSON file:
+
+```js
+const { decryptFromFile } = require("gitops-secrets");
+
+// Internally managed storage
+const secrets = decryptFromFile();
+
+// Custom Path
+const secrets = decryptFromFile(".secrets.enc.json");
+
+// Optionally merge secrets into environment variables
+secrets.populateEnv();
+```
+
+### JS Module
+
+The JS module format suits restricted environments where reading static files is problematic and depending upon the platform, building with a custom path may be required.
+
+To encrypt secrets to a JS module:
+
+```js
+const secrets = require("gitops-secrets");
+
+async function main() {
+  const payload = await secrets.providers.doppler.fetch();
+
+  // Internally managed storage
+  secrets.build(payload);
+
+  // Custom path for restrictive environments
+  secrets.build(payload, { path: "lib/secrets.js" });
+}
+
+main();
+```
+
+Then to decrypt secrets from a JS module, you can rely on internally managed storage:
+
+```js
+const { loadSecrets } = require("gitops-secrets");
+const secrets = loadSecrets();
+
+// Optionally merge secrets into environment variables
+secrets.populateEnv();
+```
+
+Or import directly from the generated JS module:
+
+```js
+const { loadSecrets } = require("../lib/secrets");
+const secrets = loadSecrets();
+
+// Optionally merge secrets into environment variables
+secrets.populateEnv();
+```
+
+### Getting Started
+
+We recommend checking out the [Working around Vercel’s 4KB Environment Variables Limit for Node.js with GitOps Secrets article](https://hashnode.com/preview/623404babef4c71aa6f0d65e) which takes you through the entire process step-by-step.
+
+## Examples
+
+Take a look at the [Vercel GitOps Secrets Next.js sample repository](https://github.com/DopplerUniversity/vercel-gitops-secrets-nextjs) and deploy to Vercel to see it in action.
+
+## Contributing
+
+As this package is still in developer preview, a huge contribution you can make is simply testing this with your preferred framework and serverless provider as we'd love your feedback!
+
+You can get support in the [Doppler community forum](https://community.doppler.com/), find us on [Twitter](https://twitter.com/doppler), and for bugs or feature requests, [create an issue](https://github.com/DopplerUniversity/gitops-secrets-nodejs/issues) on this repository.
+
+We'd also love to see the number of providers grow and you can check out our [contributing guide](CONTRIBUTING.md) to get started.
```

Comment:
This link broke for me -- do we plan to add a `CONTRIBUTING.md` file in another PR?

Reply:
Disregard; I see that you added it in #5 👍